modules/tinymce/src/core/main/ts/content/InsertContentImpl.ts

```diff
@@ -22,6 +22,19 @@ import * as SelectionUtils from '../selection/SelectionUtils';
 import { InsertContentDetails } from './ContentTypes';
 import * as InsertList from './InsertList';
 
+const wrappedElements = [ 'pre' ];
+
+const shouldFlatten = (fragment: AstNode, parentNode: Element) => {
+  const currentNode = fragment.firstChild;
+
+  const isAFlattenableTag = Arr.contains(wrappedElements, currentNode.name);
+  const isPastingInTheSameTag = currentNode.name === parentNode.tagName.toLowerCase();
+  // TINY-7719: last children is the bookmarkHtml attached by the function insertHtmlAtCaret
```

Comment:
```suggestion
  // The last children is the bookmarkHtml attached by the function insertHtmlAtCaret
```
This logic/comment has no relation to the Jira so we shouldn't be mentioning it here.

Reply:
done